src/sentry/constants.py

```diff
@@ -573,22 +573,22 @@ def as_str_choices(cls) -> Sequence[Tuple[str, str]]:
         )
 
     @classmethod
-    def as_str(cls, integer: int) -> Optional[str]:
+    def as_str(cls, integer: int) -> str:
         if integer == cls.ISSUES_BY_TAG:
             return cls.ISSUES_BY_TAG_STR
         elif integer == cls.DISCOVER:
             return cls.DISCOVER_STR
         else:
```

Comment:
just checking there's nowhere that depends on this return None behavior?

Reply:
Nothing that's covered by tests or type-checking, except [the one place](https://github.com/getsentry/sentry/pull/57883/files#diff-2f2e81587ebbd4dfe5be0c183fed2a1442a39d9787620df91fcfc5a34a276541R31-R33) I've changed.